modules/sno-du-conn-configuring-logging-locally-and-forwarding.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-logging-locally-and-forwarding_{context}"]
+= Configuring logging locally and forwarding
+
+To be able to debug a single node distributed unit (DU), logs need to be stored for further
+analysis.
+
+.Procedure
+
+* Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:
++
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging <1>
+ metadata:
```

Comment:
changed

Reply:
@StephenJamesSmith FYI, in case you didn't see the update on #team-docs, our ocp-team netlify licence has been revoked. you'll need to provide a manual build for your PR. See here for details: https://docs.engineering.redhat.com/display/TELCODOCS/Configuring+a+local+netlify+replacement+build

Reply:
@yliu Will update very soon.